bin/ch/WScriptJsrt.cpp

```diff
@@ -808,8 +808,10 @@ WScriptJsrt::CallbackMessage::~CallbackMessage()
     {
         WScriptJsrt::PrintException("", JsErrorScriptException);
     }
+CLANG_WNO_BEGIN("-Wunused-variable")
     JsErrorCode errorCode = ChakraRTInterface::JsRelease(m_function, nullptr);
```

Comment:
No we can't use Assert(...) here, which strips the code away on release build.

Why don't we use an "UNUSED" macro solution like here: https://github.com/nodejs/node-chakracore/pull/83/commits/ae1b9d49277dd58684b38c64fdb145874ecfac80#diff-6a427a4aebd067397b4135287236dae5R41